platforms/tktrex/backend/config/settings.json

```diff
@@ -5,7 +5,7 @@
     "htmls": "htmls",
     "metadata": "metadata",
     "full": "full",
-    "emails": "emails2",
+    "emails": "emails",
     "experiments": "experiments2"
```

Comment:
This should be  `"experiments": "experiments"`

Reply:
fixed